extensions/amp-tiktok/0.1/amp-tiktok.js

```diff
@@ -71,7 +76,9 @@ export class AmpTiktok extends AMP.BaseElement {
         'pointer-events',
       ]);
       this.iframe_.removeAttribute('aria-hidden');
-      this.iframe_.setAttribute('aria-label', 'Tiktok');
+      if (!this.iframe_.getAttribute('aria-title')) {
+        this.iframe_.setAttribute('aria-title', 'TikTok');
+      }
```

Comment:
`title` is the wcag attribute, which can be written as the iframe property instead of via `setAttribute`
```suggestion
     iframe.title = this.element.title || 'TikTok';
```